code/modules/projectiles/projectile/bullets.dm

```diff
@@ -59,6 +59,10 @@
 		if(D.glass) chance *= 2
 	else if(istype(A, /obj/structure/girder))
 		chance = 100
+	else if(istype(A, /obj/structure/low_wall))
+		chance = round(penetrating * armor_penetration * 2 / 150 * 180) // hardcoded, value is same as steel wall, will have to be changed once low walls have integrity
```

Comment:
don't hardcode , we have material integrity for a reason.

Reply:
low walls don't